### ISSUE TITLE
Can't use lib as standard python package

### ISSUE THREAD

**Issue by giacomobartoloni** [etrombly/RFM69#18]
Is there a way to use this lib as a normal python package?

**Comment by etrombly** [comment]
Yes, I never got around to making an installer for it. I'll work on one, may take me a little bit because I'm working on other stuff now.

**Comment by grote** [comment]
Isn't it just a matter if adding a file like this one?
https://github.com/russss/rfm69-python/blob/master/setup.py

Probably needs another name now, as rfm69 is taken.
